tests/test_datashader.py

```diff
@@ -1,13 +1,17 @@
 import unittest
 
-import numpy as np
-import pandas as pd
-import datashader as ds
-import datashader.transfer_functions as tf
+from common import p100_exempt
 
 class TestDatashader(unittest.TestCase):
-    # based on https://github.com/pyviz/datashader/blob/master/datashader/tests/test_pipeline.py
+
+    @p100_exempt
```

Comment:
Can we comment *why* certain tests are p100 example?

Reply:
d.